engine/OthelloEngine.py

```diff
@@ -31,11 +31,20 @@ def __init__(self, white_team_file, black_team_file, output_file, n = 8, time_li
       # TODO Task 4 Below:
       # Initiaize white_team_file and black_team_file's AIs
       # (might want to refer to http://www.blog.pythonlibrary.org/2012/07/31/advanced-python-how-to-dynamically-load-modules-or-classes/)
-      self.white_team = None
-      self.black_team = None
+      w_module = __import__(white_team_file)
+      b_module = __import__(black_team_file)
+      self.white_team = getattr(w_module, "Othello_AI")
+      self.black_team = getattr(b_module, "Othello_AI")
```

Comment:
Could you also instantiate an object from these classes? white_team and black_team should be objects ready to use. (Also, I accidentally left a function called init instead of __init__ in OthelloInterface.py, so if you could change that too, that would be great)

Reply:
Do you mean call their respective __init__ methods?

Reply:
Yes, I think it should be something similar to:
```
self.white_team = (getattr(w_module, "Othello_AI"))('W', n, time_limit)
```

Reply:
Done